test/smart_city/test_data_generator_test.exs

```diff
@@ -7,19 +7,25 @@ defmodule SmartCity.TestDataGeneratorTest do
     assert match?(%SmartCity.Dataset{}, TDG.create_dataset(%{}))
   end
 
+  test "create_dataset/1 creates a valid dataset with an overridden organization" do
+    org_id = "fake_org_id"
+    test_dataset = TDG.create_dataset(%{organization_id: org_id})
+    assert test_dataset.organization_id == org_id
+    assert match?(%SmartCity.Dataset{}, TDG.create_dataset(%{}))
+  end
+
   test "create_dataset/1 makes a valid system name :|" do
     dataset = TDG.create_dataset(%{})
 
-    assert dataset.technical.systemName ==
-             "#{dataset.technical.orgName}__#{dataset.technical.dataName}"
+    assert dataset.technical.systemName =~
+             "#{dataset.technical.dataName}"
```

Comment:
Should this test be asserting for `<wildcard>__#{dataset.technocal.dataName}` or at least validate that the __ comes before the dataName?

Reply:
Added a validation that the __ is in the correct place! (Didn't add the wildcard validation because we weren't quite sure how that would work.)